### PR TITLE
858353_Prepare UG document for MAUI Sunburst chart control

### PR DESCRIPTION
**Pull request by EzhilarasanElangovan31**
**Description**

Added UG document for MAUI Sunburst chart control

### CODE REVIEW

File: MAUI/Sunburst-Chart/Getting-Started.md [hunk 1]
```diff
@@ -0,0 +1,395 @@
...
+
+    this.BindingContext = new ViewModel();
+
+{% endhighlight %}
+
+{% endtabs %} 
+
+## Populate chart with data
+
+ Binding `DataSource` to the Sunburst chart [ItemsSource]() property from its BindingContext to create our own Sunburst chart.
```
Comment (VimalaThirumalaiKumarSF3739) [inline MAUI/Sunburst-Chart/Getting-Started.md:191]: Include the content for adding SunburstHierarchicalLevel and ValueMemberPath

File: MAUI/Sunburst-Chart/Getting-Started.md [hunk 2]
```diff
@@ -0,0 +1,395 @@
...
+            sunburst.ShowLabels = true;
+            this.Content = sunburst;
+        }
+    }
+
+{% endhighlight %}
+
+{% endtabs %}
+
+![SunburstChart chart in .NET MAUI Chart](Getting-Started_Images/maui_sunburst_chart.png)
```
Comment (VimalaThirumalaiKumarSF3739) [inline MAUI/Sunburst-Chart/Getting-Started.md:393]: Modify as - Sunburst Chart in .NET MAUI

File: MAUI/Sunburst-Chart/Getting-Started.md (hunk 1 above)
Comment (EzhilarasanElangovan31) [reply MAUI/Sunburst-Chart/Getting-Started.md:191]: content Included

File: MAUI/Sunburst-Chart/Getting-Started.md (hunk 2 above)
Comment (EzhilarasanElangovan31) [reply MAUI/Sunburst-Chart/Getting-Started.md:393]: Modified

**Review by Saravanan-Madhesh — APPROVED** [review]
Approving and will check in staging.

**Pull request merged**